projects/plugins/jetpack/_inc/blogging-prompts.php

```diff
@@ -32,7 +32,7 @@ function jetpack_blogging_prompts_add_meta_data( $keys ) {
  * @return void
  */
 function jetpack_setup_blogging_prompt_response( $post_id ) {
-	// phpcs:ignore WordPress.Security.NonceVerification.Recommended
+	// phpcs:ignore WordPress.Security.NonceVerification.Recommended -- Clicking a prompt response link can happen from notifications, Calypso, wp-admin, etc and only setups up a response post (tag, meta, prompt text); the user must take action to actually publish the post.
```

Comment:
Should we worry about the list of "draft" posts getting filled up by an attacker?

Reply:
It's a fair question! In practice, I'm personally not worried--it seems like a lot of effort (or patience) to get a user to open the answer_prompt link enough times to be concerning. I think there are probably easier and lower effort attacks.

For me the desire to make answering a writing prompt as seamless as possible from multiple contexts outweighs the (arguably) minimal security benefit of a nonce in this particular case.

Reply:
Fair enough. I don't know enough about this to second-guess the decision, I just wanted to confirm that the decision was made with relevant factors having been considered.